consensus/src/experimental/buffer_item.rs

```diff
@@ -80,7 +78,7 @@ fn aggregate_commit_proof(
     validator: &ValidatorVerifier,
 ) -> LedgerInfoWithSignatures {
     let aggregated_sig = validator
-        .aggregate_and_verify_multi_signature(verified_signatures, commit_ledger_info)
+        .generate_multi_signature(verified_signatures, commit_ledger_info)
```

Comment:
As before, I find the `generate_` terminology awkward here.

Reply:
I am open to suggestions but this is the best name I could find. I can use `aggregate_multi_signature` as well but I think that will be more confusing :)

Reply:
Is this a multisignature (on the same m) or an aggregate signature (on different m)?

If so, either `aggregate_multi_signature` or `aggregate_signature` are fine, no?

Reply:
This is a multisignature of same message, we can change it to `aggregate_multi_signature` but I have a similar method `generate_aggregate_signatre` and to be consistent, I will have to rename that to `aggregate_aggregate_signature`, which becomes more confusing. I think `generate_multi_signature` and `generate_aggregate_signature` is fine as they are clear and consistent. LMK what you think.

Reply:
Ah, okay!